specification/network/resource-manager/Microsoft.Network/stable/2018-06-01/serviceEndpointPolicy.json

```diff
@@ -0,0 +1,371 @@
+      "get": {
+        "tags": [
+          "serviceEndpointPolicies"
+        ],
+        "operationId": "serviceEndpointPolicies_List",
+        "description": "Gets all service endpoint Policies in a resource group.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. The operation returns a list of ServiceEndpointPolicy resources.",
+            "schema": {
+              "$ref": "#/definitions/ServiceEndpointPolicyListResult"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List resource group service endpoint policies": { "$ref": "./examples/ServiceEndpointPolicyList.json" }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ServiceEndpointPolicy": {
+      "properties": {
+        "serviceEndpointPolicyDefinitions": {
+          "type": "array",
+          "items": {  
+          "$ref": "#/definitions/serviceEndpointPolicyDefinitionsFormat",
+          },  
+          "description": "List of service endpoint definitions."
+        },
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/Resource"
+        }
+      ],
+      "description": "Service endpoint policy."
+    },
+    "ServiceEndpointPolicyListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ServiceEndpointPolicy"
+          },
+          "description": "A list of service endpoint policies that exists in a resource group."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to get the next set of results."
+        }
+      },
+      "description": "Response for ListServiceEndpointPolicies API service call."
+    },
+    "serviceEndpointPolicyDefinitionsFormat": {
+      "name": {
+        "type": "string",
+        "description": "The name of service endpoint policy definition."
+      },
+      "properties": {
+        "description": {
+          "type": "string",
+          "description": "The description of service endpoint policy definition."
+        },
+        "service": {
+          "type": "string",
+          "description": "service endpoint name."
+        },
+        "serviceResources": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "A list of service resources."
+        },
+        "provisioningState": {
```

Comment:
I understand this is usually an enum. Must be readonly.

Reply:
done